src/cogent3/parse/sequence.py

```diff
@@ -90,6 +91,52 @@ def format_from_filename(filename, format=None):
         return filename[filename.rfind(".") + 1 :]
 
 
+def load_from_json(filename, classes):
+    """Loads objects from json files.
+
+    Parameters
+    ----------
+    filename: name of the json file
+    classes: A series of the Cogent3 types, for example: (Alignment, ArrayAlignment)
+
+    """
+    assert all(
+        (isinstance(klass, type) for klass in classes)
+    ), "classes should be a series of Cogent3 types, for example: (Alignment, ArrayAlignment)"
+
```

Comment:
I think we use a string comparison for types, since the dict in the json file encodes this as a string. So let's create a set object:
```python
valid_types = {get_object_provenance(klass) for klass in classes}
```

Reply:
Hi Gavin! thanks again for your detailed instructions! 

This line of codes just checking if users input correct `classes` parameter as it should be a series of types.

Reply:
correct, but my suggestion is to create this set object after you've checked the user has passed in types (so add this code below yours)